tests/test_lexer.py

```diff
@@ -0,0 +1,277 @@
+"""Tests for lexer-related code."""
+
+import unittest
+from automata.regex.postfix import Token
+from automata.regex.postfix import LeftParen, RightParen
+from automata.regex.lexer import TokenRegistry, Lexer, LexerError
+
+
+def register_parens(lexer: Lexer) -> None:
+    lexer.register_token(lambda text: LeftParen(text), LeftParen.regexp())
+    lexer.register_token(lambda text: RightParen(text), RightParen.regexp())
+
+
+class TestTokenRegistryTestCase(unittest.TestCase):
+
+    def test_add_token(self) -> None:
+        class AToken(Token):
+            pass
+
+        class BToken(Token):
+            pass
+
+        registry: TokenRegistry = TokenRegistry()
+        assert 0 == len(registry._tokens)
```

Comment:
@eliotwrobson Can you please convert these to proper unittest assertions (e.g. `self.assertEqual`)?